@here/harp-map-controls/test/MapControlsTest.ts

```diff
@@ -93,42 +91,95 @@ describe("MapControls", function() {
     });
 
     it("correctly updates mapView on mouse move", function() {
-        const addEventListenerStub = sandbox.stub();
         const updateStub = sandbox.stub();
-        const domElement = {
-            addEventListener: addEventListenerStub
-        } as any;
         //@ts-ignore
-        const mapControls = new MapControls({
-            renderer: { domElement } as any,
+        const controls = new MapControls({
+            renderer: { domElement: { addEventListener: sandbox.stub() } } as any,
             update: updateStub
         });
-        sandbox.stub(mapControls, "dispatchEvent");
-        sandbox.stub(mapControls, "getPointerPosition").returns({ x: 0, y: 0 });
+        sandbox.stub(controls, "dispatchEvent");
+        sandbox.stub(controls, "getPointerPosition").returns({ x: 0, y: 0 });
 
         expect(updateStub.callCount).to.be.equal(0);
-        mapControls.mouseMove({ preventDefault: sandbox.stub(), stopPropagation: sandbox.stub() });
+        controls.mouseMove({ preventDefault: sandbox.stub(), stopPropagation: sandbox.stub() });
         expect(updateStub.callCount).to.be.equal(1);
     });
 
     it("correctly updates mapView on touch move", function() {
         const updateStub = sandbox.stub();
         //@ts-ignore
-        const mapControls = new MapControls({
+        const controls = new MapControls({
             renderer: { domElement: { addEventListener: sandbox.stub() } as any } as any,
             update: updateStub
         });
-        mapControls.m_touchState.touches = { length: 5 };
-        sandbox.stub(mapControls, "updateTouches");
-        sandbox.stub(mapControls, "dispatchEvent");
-        sandbox.stub(mapControls, "getPointerPosition").returns({ x: 0, y: 0 });
+        controls.m_touchState.touches = { length: 5 };
+        sandbox.stub(controls, "updateTouches");
+        sandbox.stub(controls, "dispatchEvent");
+        sandbox.stub(controls, "getPointerPosition").returns({ x: 0, y: 0 });
 
         expect(updateStub.callCount).to.be.equal(0);
-        mapControls.touchMove({
+        controls.touchMove({
             touches: [],
             preventDefault: sandbox.stub(),
             stopPropagation: sandbox.stub()
         });
         expect(updateStub.callCount).to.be.equal(1);
     });
+
+    describe("zoomOnTargetPosition", function() {
+        const elevationProvider = ({} as any) as ElevationProvider;
+
+        function resetCamera() {
+            const target = GeoCoordinates.fromDegrees(0, 0);
+            const heading = 0;
+            const tilt = 0;
```

Comment:
Could you add some test with tilt angle "enabled".

Reply:
done